ArduPlane/quadplane.cpp

```diff
@@ -1423,12 +1417,12 @@ void SLT_Transition::update()
         // the quad should provide some assistance to the plane
         quadplane.assisted_flight = true;
         // update transition state for vehicles using airspeed wait
-        if (transition_state != TRANSITION_AIRSPEED_WAIT) {
+        if (transition_state != TRANSITION_AIRSPEED_WAIT && !in_forced_transition) {
```

Comment:
This is a very subtle change in behavior. 

I think it should be:
```
        if (!in_forced_transition) {
            if (transition_state != TRANSITION_AIRSPEED_WAIT) {
                gcs().send_text(MAV_SEVERITY_INFO, "Transition started airspeed %.1f", (double)aspeed);
            }
            transition_state = TRANSITION_AIRSPEED_WAIT;
            if (transition_start_ms == 0) {
                transition_start_ms = now;
            }
        }
```
The key change is the resting of `transition_start_ms = now;` while still in `TRANSITION_AIRSPEED_WAIT`